src/Css/CssDocument.php

```diff
@@ -115,7 +115,7 @@ public function renderNonConditionalAtRules(): string
     }
 
     /**
-     * @param array<array-key, string> $allowedMediaTypes
+     * @param array<array-key, non-empty-string> $allowedMediaTypes
```

Comment:
We can also change this one, which is only called by `getStyleRulesData()`.

```suggestion
     * @param list<non-empty-string> $allowedMediaTypes
```